source/Calamari.Shared/Deployment/Conventions/StageScriptPackagesConvention.cs

```diff
@@ -1,100 +1,101 @@
 ﻿using System.IO;
-using System.Linq;
 using Calamari.Common.Commands;
 using Calamari.Common.Features.Packages;
 using Calamari.Common.Plumbing.FileSystem;
 using Calamari.Common.Plumbing.Logging;
 using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment.Conventions.DependencyVariablesStrategies;
 
 namespace Calamari.Deployment.Conventions
 {
     public class StageScriptPackagesConvention : IInstallConvention
```

Comment:
This class is exactly the same for packages and Git dependencies, except the variable names.